src/app/app.module.ts

```diff
@@ -1,18 +1,26 @@
 import { BrowserModule } from '@angular/platform-browser';
 import { NgModule } from '@angular/core';
+import { FormsModule } from '@angular/forms';
+import { HttpModule } from '@angular/http';
+import { NgbModule } from '@ng-bootstrap/ng-bootstrap';
 
 import { AppRoutingModule } from './app-routing.module';
 import { AppComponent } from './app.component';
+import { RestComponent } from './rest/rest.component';
 
 @NgModule({
   declarations: [
-    AppComponent
+    AppComponent,
+    RestComponent
```

Comment:
where you using this component RestComponent?

Reply:
deleted